examples/dnf-update.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+dnf upgrade -y -x kube*
+touch /var/run/reboot-required
```

Comment:
Are we sure that we want to require a reboot after every `dnf upgrade`? Does `dnf` not do that automatically depending on what updates it pulls down (in other words, I assume there are some upgrades that can be applied without a reboot)?

Reply:
During testing, I needed to reboot to get the updates to apply.